themes/aframe/source/css/secondary.styl

```diff
@@ -172,6 +172,49 @@ a
 .section-subnav
     margin-bottom 10px
 
+.copy .post__heading
+    margin-bottom 0
+
+.post__projects
+    display flex
+    flex-wrap wrap
+    margin-bottom 15px
+    margin-left: -1px
+    a
+        background-position center
+        background-repeat no-repeat
+        background-size cover
+        border 1px solid transparent
+        height 160px
```

Comment:
`160px` seems a tad tall for some of the images:

![image](https://cloud.githubusercontent.com/assets/203725/14481585/40aa0d72-00e9-11e6-8a07-74787d58f200.png)